src/SALib/analyze/morris.py

```diff
@@ -31,9 +32,17 @@ def analyze(
 
     - ``mu`` metric indicates the mean of the distribution
     - ``mu_star`` metric indicates the mean of the distribution of absolute
-        values
+      values
     - ``sigma`` is the standard deviation of the distribution
 
+    When ``scaled`` is True, the elementary effects are scaled by the ratio of
+    standard deviation of ``X`` and ``Y`` according to [3]. When using this
+    option it is important to ensure that ``X`` contains the actual values
+    passed into the model, as the elementary effects are divided by the
+    step calculated from ``X`` rather than using `delta` which is calculated from
+    the number of levels used in the sample. This could be the case if you
+    perform post-processing on the values before passing them to the model.
+
```

Comment:
It would also be good to add some information on when it is desirable to use the scaled, non-dimensional, sensitivity measure (e.g., avoids the dependency between factor rankings and the units of the factors)